app/sections/view-client-application-types/template.njk

```diff
@@ -9,93 +9,102 @@
+                      }
+                    },
+                    'native-mobile-connection-details': {
+                      answers: {
+                        'minimum-connection-speed': '1Mbps',
+                        'connection-types': ['GPRS','3G','4G','Wifi'],
+                        'connection-requirements-description': 'Average data usage will vary depending on application activity.'
+                      }
+                    },
+                    'native-mobile-third-party': {
+                      answers: {
+                        'third-party-components': 'The application supports and requires an authenticator on each device the application is installed. You will need a software-based authenticator that implements a two-step verification service.',
+                        'device-capabilities': 'In order to use our file hosting services, the application will require permission to access device storage.'
+                      }
+                    },
+                    'native-mobile-hardware-requirements': {
+                      answers: {
+                        'hardware-requirements': 'To fully utilise our print functionality within the application, you will need a WiFi or Bluetooth connected printer to connect and print documents straight from the device.'
+                      }
+                    },
+                    'native-mobile-additional-information': {
+                      answers: {
+                        'additional-information': 'It is possible that it may install on other platforms or versions not listed in this section. However, support is limited to systems that meet the minimum requirements.'
+                      }
                     }
-                  },
-                  'native-mobile-hardware-requirements': {
-                    answers: {
-                      'hardware-requirements': 'To fully utilise our print functionality within the application, you will need a WiFi or Bluetooth connected printer to connect and print documents straight from the device.'
-                    }
-                  },
-                  'native-mobile-additional-information': {
-                    answers: {
-                      'additional-information': 'It is possible that it may install on other platforms or versions not listed in this section. However, support is limited to systems that meet the minimum requirements.'
+                  }
+                },
+                'native-desktop': {
+                  sections: {
+                    'native-desktop-operating-systems': {
+                      answers: {
+                        'operating-systems-description': 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.'
```

Comment:
We've always used some kind of semi descriptive test data above. I would do the same here. Even if it is something like `Some operating system description data for native desktop`